perllib/FixMyStreet/App/Controller/My.pm

```diff
@@ -234,6 +234,35 @@ sub by_shortlisted {
     }
 }
 
+sub anonymize : Path('anonymize') {
```

Comment:
Should this controller require an HTTP `POST`, as it modifies (potentially lots of) reports?